examples/tutorials/Working_With_Splitters.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>Could you resolve the errors at the top and push the notebook again</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3837/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>